Library/Homebrew/test/rubocops/unless_multiple_conditions_spec.rb

```diff
@@ -0,0 +1,72 @@
+# typed: false
+# frozen_string_literal: true
+
+require "rubocops/unless_multiple_conditions"
+
+describe RuboCop::Cop::Style::UnlessMultipleConditions do
+  subject(:cop) { described_class.new }
+
+  it "reports an offense when using `unless` with multiple `and` conditions" do
+    expect_offense <<~RUBY
+      unless foo && bar
+      ^^^^^^^^^^^^^^^^^ Avoid using `unless` with multiple conditions.
+        something
+      end
+    RUBY
+
+    expect_offense <<~RUBY
+      something unless foo && bar
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Avoid using `unless` with multiple conditions.
+    RUBY
+  end
+
+  it "reports an offense when using `unless` with multiple `or` conditions" do
+    expect_offense <<~RUBY
+      unless foo || bar
+      ^^^^^^^^^^^^^^^^^ Avoid using `unless` with multiple conditions.
+        something
+      end
+    RUBY
+
+    expect_offense <<~RUBY
+      something unless foo || bar
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Avoid using `unless` with multiple conditions.
```

Comment:
Which one do prefer?

```rb
something unless foo || bar
^^^^^^^^^^^^^^^^^^^^^^^^^^^ Avoid using `unless` with multiple conditions.
```

```rb
something unless foo || bar
                 ^^^^^^^^^^ Avoid using `unless` with multiple conditions.
```

As far as I can tell, it's not possible to do this:

```rb
something unless foo || bar
          ^^^^^^^^^^^^^^^^^ Avoid using `unless` with multiple conditions.
```

Reply:
I like the last one. The issue is with the `unless` and the condition, but not with the `something`.

This would also set up well for an autocorrect which I think should be relatively easy to do here (I say without trying). Following [De Morgan's laws](https://en.wikipedia.org/wiki/De_Morgan%27s_laws), the following pairs are equivalent:

```ruby
something unless foo || bar
something if !foo && !bar
```

```ruby
something unless foo && bar
something if !foo || !bar
```

```ruby
something unless foo || bar || baz
something if !foo && !bar && !baz
```

Essentially, flip `unless` to `if`, negate each condition, and change `||` to `&&` (and vice-versa).